libpod/reset.go

```diff
@@ -157,7 +157,13 @@ func (r *Runtime) reset(ctx context.Context) error {
 		}
 	}
 
-	xdgRuntimeDir := filepath.Clean(os.Getenv("XDG_RUNTIME_DIR"))
+	xdgRuntimeDir := os.Getenv("XDG_RUNTIME_DIR")
+	if xdgRuntimeDir != "" {
```

Comment:
Again, I think this is misleading because `XDG` is always, always going to be set very early on by `config` code.

Reply:
There is no change here if XDG_RUNTIME_DIR is not set or is set to "", Current code would not fail if XDG_RUNTIME_DIR=="" new code would, so I add his check.